config/initializers/rack_attack.rb

```diff
@@ -3,6 +3,8 @@
 
 # Is this likely to affect the accessible request form if a lot of errors?
 
+Rack::Attack.cache.store = Redis.new(url: ENV["REDIS_URL"]) if ENV["REDIS_URL"]
```

Comment:
what happens if `ENV["REDIS_URL"]` is not set? is this OK with a `nil` or do we need a default somewhere?

Reply:
Good question. I suspect it'll try to use the existing rails cache (probably filecache?). 

Without redis, I think we have to throw up a warning in the log and disable rate-limiting.